docs/image/index.rst

```diff
@@ -12,25 +12,39 @@ Introduction
 ============
 
 `gammapy.image` contains data classes and methods for image based analysis
-of gamma-ray data. Currently it includes multi purpose image processing
-methods as well. The goal longterm goal is to contribute most of these methods
-to `scipy.ndimage`, `scikit-image`_, `astropy`_ or `photutils`_, and to only keep
-gamma-ray analysis specific functionality here.
+of gamma-ray data.
+
 
 Getting Started
 ===============
 
-Most of the functions in this module have objects of type `numpy.array`
-or an `astropy.io.fits.ImageHDU` or `astropy.io.fits.PrimaryHDU`
-as input and output:
+The central data structure in `gammapy.image` is the `gammapy.image.SkyImage`
+class, which combines the raw data with WCS information, FITS I/O functionality
+and many other methods, that allow easy handling, processing and plotting of
+image based data. Here is a first example:
 
 .. code-block:: python
 
-   >>> from gammapy.datasets import poisson_stats_image
-   >>> from gammapy.image import lookup
-   >>> image = poisson_stats_image() # image is a 2D numpy array
-   >>> lookup(image, 42, 44, world=False)
-   3.0
+   >>> from gammapy.datasets import gammapy_extra
+   >>> from gammapy.image import SkyImage
+   >>> filename = gammapy_extra.filename('datasets/fermi_2fhl/fermi_2fhl_vela.fits.gz')
+   >>> skyimage = SkyImage.read(filename, ext=2)
```

Comment:
@OlgaVorokh and I have discussed `image` vs `skyimage` vs `sky_image` and decided to put just `image` almost everywhere in the code, because we pretty much only have one type of image: sky images.
My suggestion would be to also just use plain `image` as variable name everywhere in the docs as default.